website/app-templates/smarty/pages/user_recent_moves.tpl

```diff
@@ -24,7 +24,7 @@
                 </thead>
                 <tbody>
                     {foreach from=$moves.subset item=item}
```

Comment:
I ve seen that case elsewhere and opened an issue on `ikez/cortex` repo about it. IIRC we solved it differently. Grep for subset in other templates

Reply:
I dont remember why I update this user dedicated tpl, revert my upd